ocaml/libs/tracing/tracing.ml

```diff
@@ -389,12 +403,15 @@ module Spans = struct
         copy
     )
 
-  let dump () = Hashtbl.(copy spans, Hashtbl.copy finished_spans)
+  let dump () =
+    Xapi_stdext_threads.Threadext.Mutex.execute lock (fun () ->
+        Hashtbl.(copy spans, Hashtbl.copy finished_spans)
+    )
 
   module GC = struct
     let lock = Mutex.create ()
 
-    let span_timeout = ref 86400.
+    let span_timeout = Atomic.make 86400.
```

Comment:
Is this a magic number? Add a comment that explains what this is in more common terms.

Reply:
Added a comment.